extensions/objects/templates/ImageWithRegionsEditor.js

```diff
@@ -233,6 +290,14 @@ oppia.directive('imageWithRegionsEditor', [
           }
           $scope.userIsCurrentlyDrawing = false;
           $scope.userIsCurrentlyDragging = false;
+          $scope.userIsCurrentlyResizing = false;
+          if ($scope.outOfRegion) {
+            $scope.xDirection = 0;
+            $scope.yDirection = 0;
+          }
```

Comment:
The var outofRegion is necessary.
Consider the following behaviour: While resizing in the upper direction if you move horizontally in the left or the right direction the you can move out of the rectangular region. The in that case the cursor needs to be reset and this variable makes sure the same.

Actually using the margin part I was able to make sure the cursor never needs to be reset when resized except for the above behaviour. I'll make a commit with the